homeassistant/components/twinkly/light.py

```diff
@@ -53,6 +54,10 @@ def __init__(
         self._hass = hass
         self._conf = conf
 
+        self._attr_supported_color_modes = {COLOR_MODE_RGBW}
+        self._attr_color_mode = COLOR_MODE_RGBW
+        self._attr_rgbw_color = (255, 255, 255, 0)
```

Comment:
Well not all support RGBW, we should handle it dynamic and only show the support if they are present over an property

Reply:
Yep, you're right. It got burried a bit over the numerous comments. It's on my list to adjust.